app/src/github_runner_image_builder/templates/cloud-init.sh.j2

```diff
@@ -36,6 +46,18 @@ EOF
 function install_apt_packages() {
     local packages="$1"
     local hwe_version="$2"
+
+    # The gh package (GitHub CLI application) is not in the APT repository for focal.
+    # For focal, the apt repository of GitHub is added.
+    if [ $RELEASE == focal ]; then
```

Comment:
woud this need to be "focal"?

Reply:
Updated.
I should work due to how bash script works. However, `"focal"` would be better stylistically.